e2e/_suites/ingest-manager/features/ingest-manager.feature

```diff
@@ -0,0 +1,46 @@
+@ingest
+Feature: Enable Fleet user and create initial Kibana setup
+
+Scenario: Enrolling an agent
+  Given there is a "Fleet" user in Kibana
+    And the "Fleet" Kibana setup has been created
```

Comment:
I'd group this two, which seems to me as an always-together pair, into one step:

```gherkin
Given "Fleet" is configured in Kibana
```

Reply:
note, the above api_key relates to an ephemeral instance of Kibana and it was anonymized / changed anyways (it was not a valid key).